### PR TITLE
SNMN-268 Set up Depth2HHA-python build pipeline

### PR DESCRIPTION
**Pull request by piairoeduardo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 __pycache__
 .DS_Store
+
+# PyCharm
+/.idea/*
```
Comment (dribeiro09) [inline .gitignore:5]: Can we add vscode as well?
```
# VSCode
.vscode
```

**Review by dribeiro09 — APPROVED** [review]

**Review by bmframos — CHANGES_REQUESTED** [review]
The package is not being correctly built, [see in gemfury](https://manage.fury.io/dashboard/deeperinsights/package/pkg_9IS4v/version/ver_1wY8aS), the full util module is missing, so the package is actually empty.

I remember that at the time there were some changes that needed to be done, but they were apparently not updated on this repo 😞 
I think adding adding an `__init__.py` to utils would fix it, but the [way we are importing it](https://github.com/skimit/markerless-navigation/blob/master/markerless_navigation/__init__.py#L1-L2) in S&N, I think the correct solution is to move all the utils to the parent folder and create the `__init__.py` there.

**Comment by piairoeduardo** [comment]
@bmframos Added the file `__init__.py` to the `utils` folder.

**Comment by bmframos** [comment]
> @bmframos Added the file `__init__.py` to the `utils` folder.

Everything is getting getting imported now, can we just make 1 small change and rename `utils` package to `depth2hha`, so that it follows the structure of imports that we use on S&N. Otherwise the package is being imported as `utils` module
Screenshot 2021-09-14 at 12 11 13

**Comment by piairoeduardo** [comment]
The folder `utils` was renamed to  `depth2hha`.

**Review by bmframos — APPROVED** [review]
Everything looks good! Just the package version hasn't bumped from 0.0.18 after the last commit

**Pull request merged**